_posts/2017-08-08-migrating-existing-datastores.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+id: migrating-existing-datastores
+title: "Migrating Existing Datastores"
```

Comment:
I think from the Google doc, the title is "So you are plan to migrate from existing datastore. Here's what comes next", unless you all agreed to change it?